filament/include/filament/Texture.h

```diff
@@ -236,6 +236,14 @@ class UTILS_PUBLIC Texture : public FilamentAPI {
          */
         Builder& import(intptr_t id) noexcept;
 
+        /**
+         * Specifies the sample count of the texture
+         *
+         * @param sampleCount sample count
+         * @return This Builder, for chaining calls.
+         */
+        Builder& samples(uint8_t sampleCount) noexcept;
```

Comment:
super nit: I'd place this declaration between `sampler()` and `format()`, as it already is in `Texture.cpp`

Reply:
fixed in https://github.com/shapr3d/filament/pull/67/commits/917a91248e3bd5fdf063c55e3dcdbf473fc76d29